console/src/main/java/com/alibaba/nacos/console/security/nacos/roles/NacosRoleServiceImpl.java

```diff
@@ -158,7 +158,7 @@ public boolean hasPermission(String username, Permission permission) {
     
     public List<RoleInfo> getRoles(String username) {
         List<RoleInfo> roleInfoList = roleInfoMap.get(username);
-        if (!authConfigs.isCachingEnabled()) {
+        if (!authConfigs.isCachingEnabled() || roleInfoList == null) {
```

Comment:
Why add this？there is not null judgement in 163.

Reply:
If cache missed, try to find it in database

Reply:
I think it is other change, not relative with your fixed issue.

If you want to change this. Please open a new issue to discuss.

This PR should remove them.